doc/developer-guide/api/functions/TSContScheduleOnEntirePool.en.rst

```diff
@@ -0,0 +1,60 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed
+   with this work for additional information regarding copyright
+   ownership.  The ASF licenses this file to you under the Apache
+   License, Version 2.0 (the "License"); you may not use this file
+   except in compliance with the License.  You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied.  See the License for the specific language governing
+   permissions and limitations under the License.
+
+.. include:: ../../../common.defs
+
+.. default-domain:: c
+
+TSContScheduleOnEntirePool
+**************************
+
+Synopsis
+========
+
+.. code-block:: cpp
+
+    #include <ts/ts.h>
+
+.. function:: std::vector<TSAction> TSContScheduleOnEntirePool(TSCont contp, TSHRTime timeout, TSThreadPool tp)
```

Comment:
Timeout seems wrong name here. It's a delay, right ? I got confused here because the "every" version also uses both timeout and delay "randomly"? Maybe the "every" shouldn't be using delay, but rather "interval" ?

Reply:
I'm just following the naming convention for the other `TSContSchedule` APIs. I'll update the email in the mailing list to show the changes of this PR.

The timeout is kind of like a delay, but it could also be a negative number for negative queue events.

Reply:
I thought we got rid of the negative queue.

Reply:
pretty sure it's still here? https://github.com/apache/trafficserver/blob/0d14262072fd4d8543750f764d8563ce4dff9837/src/iocore/eventsystem/UnixEThread.cc#L241